check

```diff
@@ -496,7 +496,7 @@ _run_test() {
 			if [[ ! -v SKIP_REASON ]]; then
 				unset_skip_reason=1
 				if (( !CAN_BE_ZONED )) && _test_dev_is_zoned; then
-					SKIP_REASON="${TEST_DEV} is a zoned block device"
+					_skip_reasons_append "${TEST_DEV} is a zoned block device"
```

Comment:
> Hi @zhijianli88 Thanks for rebasing.
> 
> I suggest to add note to the commit message of the 1st commit which describes that "SKIP_REASONS=" substitution should be replaced with the new function. It will be a kind of announcement for the blktests contributors.

Added

> 
> The 2nd commit message title has "common:" prefix, but this is not exactly correct. I suggest "common,tests: " as the prefix to indicate that it touches files in both directories "common" and "tests".

Fixed